cam_unit_test_batch.sh

```diff
@@ -0,0 +1,2 @@
+/bin/python /home/brock/Desktop/rr/capstone-project-rear-rider/rear_rider_device/cam_stream_unit_test.py
+/bin/python /home/brock/Desktop/rr/capstone-project-rear-rider/rear_rider_device/cam_unit_test.py
```

Comment:
Can you update these paths to be relative paths instead of absolute? Right now this script will only work for you since it starts with `/home/brock`

Reply:
For instance, remove every directory to the **left** of `rear_rider_device/`